src/main/java/seedu/address/logic/commands/customer/CustomerEditCommand.java

```diff
@@ -125,6 +137,51 @@ private static Customer createEditedCustomer(Customer customerToEdit,
                 updatedEmail, updatedAddress);
     }
 
+    /**
+     * Updates all the deliveries associated with the customer with new customer details.
+     */
+    protected static void updateDelivery(Model model, Customer editedCustomer) {
```

Comment:
Is there are a reason why this is `protected` ?

Reply:
This method is used by `CustomerEditCommandTest` to update the deliveries in the expected model

Reply:
This method is used by `CustomerEditCommandTest` to update the deliveries in the expected model